external/radiation/python/getdata.py

```diff
@@ -0,0 +1,240 @@
+    # File names for serialized random numbers in mcica_subcol
+    if me == 0:
+        sw_rand_file = os.path.join(LOOKUP_DIR, "rand2d_sw.nc")
+    else:
+        sw_rand_file = os.path.join(LOOKUP_DIR, "rand2d_tile" + str(me) + "_sw.nc")
+    lw_rand_file = os.path.join(LOOKUP_DIR, "rand2d_tile" + str(me) + "_lw.nc")
+    data_dict['sw_rand'] = xr.open_dataset(sw_rand_file)['rand2d'].values
+    data_dict['lw_rand'] = xr.open_dataset(lw_rand_file)['rand2d'].values
+    del(sw_rand_file, lw_rand_file)
+
+    return data_dict 
+
+def lw(LOOKUP_DIR):
+        ## file names needed in lwrad()
+    
+    lw_dict = {}
+    dfile = os.path.join(LOOKUP_DIR, "totplnk.nc")
+    pfile = os.path.join(LOOKUP_DIR, "radlw_ref_data.nc")
+    lw_dict['totplnk'] = xr.open_dataset(dfile)["totplnk"].values
+    lw_dict['preflog'] = xr.open_dataset(pfile)["preflog"].values
+    lw_dict['tref'] = xr.open_dataset(pfile)["tref"].values
+    lw_dict['chi_mls'] = xr.open_dataset(pfile)["chi_mls"].values
+    del(dfile, pfile)
+
+
+    ## loading data for cldprop in lwrad()
+    ds = xr.open_dataset(os.path.join(LOOKUP_DIR, "radlw_cldprlw_data.nc"))
+    lw_dict['absliq1'] = ds["absliq1"].values
+    lw_dict['absice0'] = ds["absice0"].values
+    lw_dict['absice1'] = ds["absice1"].values
+    lw_dict['absice2'] = ds["absice2"].values
+    lw_dict['absice3'] = ds["absice3"].values
+    del(ds)
+
+    ## loading data for taumol
+    varnames_bands = {1:['selfref','forref','ka_mn2','absa','absb','fracrefa','fracrefb'],
+                      2:['selfref','forref','absa','absb','fracrefa','fracrefb'],
+                      3:['selfref','forref','ka_mn2o','kb_mn2o','absa','absb','fracrefa','fracrefb'],
+                      4:['selfref','forref','absa','absb','fracrefa','fracrefb'],
+                      5:['selfref','forref','absa','absb','fracrefa','fracrefb','ka_mo3','ccl4'],
+                      6:['selfref','forref','absa','fracrefa','ka_mco2','cfc11adj','cfc12'],
+                      7:['selfref','forref','absa','absb','fracrefa','fracrefb','ka_mco2','kb_mco2'],
+                      8:['selfref','forref','absa','absb','fracrefa','fracrefb','ka_mo3','ka_mco2','kb_mco2','cfc12','ka_mn2o','kb_mn2o','cfc22adj'],
+                      9:['selfref','forref','absa','absb','fracrefa','fracrefb','ka_mn2o','kb_mn2o'],
+                      10:['selfref','forref','absa','absb','fracrefa','fracrefb'],
+                      11:['selfref','forref','absa','absb','fracrefa','fracrefb','ka_mo2','kb_mo2'],
+                      12:['selfref','forref','absa','fracrefa'],
+                      13:['selfref','forref','absa','fracrefa','fracrefb','ka_mco2','ka_mco','kb_mo3'],
+                      14:['selfref','forref','absa','absb','fracrefa','fracrefb'],
+                      15:['selfref','forref','absa','fracrefa','ka_mn2'],
+                      16:['selfref','forref','absa','absb','fracrefa','fracrefb'],
+                    }
+
+    for nband in range(1, 17): 
```

Comment:
Would probably be safer to use the keys in varname_bands
```suggestion
    for nband in varname_bands: 
```